testing/src/main/java/org/oppia/android/testing/story/StoryProgressTestHelper.kt

```diff
@@ -308,16 +308,16 @@ class StoryProgressTestHelper @Inject constructor(
   /* Test topic partial completion methods. */
 
   /**
-   * Marks the first chapter of test topic 0 story 0 as recently played. Note that this may require
+   * Marks the first chapter of test topic 0 story 0 as started not completed. Note that this may require
    * completing prerequisite chapters before the chapter can be marked as a prerequisite. See
    * [markCompletedTestTopic0Story0Exp0] for specifics on the parameters passed to this method, and
    * any other nuances.
    */
-  fun markRecentlyPlayedTestTopic0Story0Exp0(
+  fun markStartedNotCompletedTestTopic0Story0Exp0(
```

Comment:
Add description of arguments in kdoc using  `@param `

Reply:
done